test/test_eth_registrar_controller.js

```diff
@@ -106,6 +140,25 @@ contract('ETHRegistrarController', function (accounts) {
 			assert.equal((await web3.eth.getBalance(controller.address)) - balanceBefore, 28 * DAYS);
 		});
 
+		it('should permit new registrations with config', async () => {
+			var commitment = await controller.makeCommitmentWithConfig("newconfigname", registrantAccount, secret, resolver.address, registrantAccount);
```

Comment:
It would be ideal if there are test cases when either `resolver` or `addr` are empty and see they work as expected